src/darwin/Framework/CHIP/MTRSetupPayload_Internal.h

```diff
@@ -1,19 +1,30 @@
-//
-//  MTRSetupPayload_Internal.h
-//  MTR
-//
-//  Copyright © 2021 CHIP. All rights reserved.
-//
-
-#import <Foundation/Foundation.h>
+/**
+ *
```

Comment:
```suggestion
```

Reply:
Most files seem to have this superfluous blank line in the license comment. Maybe something to tidy up in bulk at some point.